packages/apps/kubernetes/templates/csi/deploy.yaml

```
@@ -134,6 +134,41 @@ spec:
             requests:
               cpu: 125m
               memory: 128Mi
+        - name: csi-snapshotter
+          args:
+          - --timeout=1m
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)